### ISSUE TITLE
SR1/SR3/SR4 - Both pages have same page title

### ISSUE THREAD

**Issue by jpinsent** [brharp/hjckrrh#124]
The page titles for All Services (A-Z) and Services by Audience both have the same page title: "Services | Computing & Communication Services". This fails Success Criterion 2.4.2, since the title cannot be used to distinguish the two pages.

Service Categories' page title is "CCS Services | Computing & Communication Services", which is distinguished from the other two views, but could be improved by being changed to "Service Categories | Computing & Communication Services".

**Comment by jpinsent** [comment]
Updating this problem. Services by Category (SR1) also has the same name as the other views. The only reason it's fine on the CCS is because it's overridden.

**Comment by jpinsent** [comment]
Services by Category, All Services (A-Z) and Services by Audience are each appropriately titled now. Tested on AODA Demo and CCS Staging. This issue has been resolved.
